src/main/java/io/cryostat/net/web/http/api/v1/RecordingsPostHandler.java

```diff
@@ -81,6 +82,8 @@
     private final Path savedRecordingsPath;
     private final Gson gson;
     private final Logger logger;
+    private final NotificationFactory notificationFactory;
+    private static final String NOTIFICATION_CATEGORY = "RecordingArchived";
```

Comment:
Oh, I think this category should be renamed - `RecordingArchived` sounds like an in-memory/active recording was copied into archives, but this handler allows users to re-upload previously archived recordings from their workstation into Cryostat.

Reply:
makes sense, I've renamed this now